packages/ds-healthcare-gov/docs/src/pages/patterns/Footer/Footer.example.jsx

```diff
@@ -45,12 +45,25 @@ class FooterExample extends React.PureComponent {
   }
 
   render() {
-    return (
-      <div>
-        <Footer initialLanguage={this.locale} />
-        {this.renderFooterToggles()}
+    const wrapperClassNames = 'ds-u-padding--1';
+    const footerTopExample = (
+      <div className="ds-l-container ds-u-text-align--right ds-u-padding-right--2 ds-u-padding-bottom--2">
+        <div className="ds-u-font-size--xl ds-u-sans ds-u-font-weight--bold ds-u-color--gray-dark ds-u-border-bottom--1 ds-u-padding-bottom--2">
+          58 DAYS
+        </div>
       </div>
     );
+    return (
+      <>
+        <div className="ds-u-margin-top--4">
+          <h6 className="preview__label">Minimal</h6>
+          <Footer initialLanguage={this.locale} />
+          <h6 className="preview__label">Footer - Top Section</h6>
+          <Footer initialLanguage={this.locale} footerTop={footerTopExample} />
+          <div className={wrapperClassNames}>{this.renderFooterToggles()}</div>
```

Comment:
Any reason you didn't just set the class directly ie `className="ds-u-padding--1"`?

Reply:
> 

There is no particular reason for using WrapperClassName, I found this in one of the DS file coding, So just thought to use same pattern.